### PR TITLE
Test API

### PR DESCRIPTION
**Pull request by StepaniukAndii**

### CODE REVIEW

File: src/test/java/TestRozetka/TestAPI.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+public class TestAPI {
+    @Test
+    public void assertUser() {
+        given()
+                .baseUri("https://reqres.in/")
```
Comment (grinred) [inline src/test/java/TestRozetka/TestAPI.java:16]: Url not Uri

Comment (StepaniukAndii) [reply src/test/java/TestRozetka/TestAPI.java:16]: sorry but it's stock method

**Comment by grinred** [comment]
merge!

**Review by grinred — APPROVED** [review]
Merge!

**Pull request merged**